docs/types/schema.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Schema
+---
+
+# Schema
+
+Every GraphQL API has a schema and that is used to define all the
+functionalities for an API. A schema is defined by passing 3
+[object types](./object-types): `Query`, `Mutation` and `Subscription`.
+
+`Mutation` and `Subscription` are optional, meanwhile `Query` has to always be
+there.
+
+This is an example of a schema defined using Strawberry:
+
+```python
+import strawberry
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def hello(self) -> str:
+        return "Hello World"
+
+
+schema = strawberry.Schema(Query)
+```
+
+## API
+
+```python
+strawberry.Schema(
+    query: Type,
+    mutation: Type = None,
+    subscription: Type = None
+    validate_queries: bool = True
+)
+```
+
+<!-- TODO: add docs on directives, types, extensions and execution context class -->
+
+Creates a GraphQL schema
+
+- `query`: The root query type
+- `mutation`: The root mutation type
+- `subscription`: The root subscription type
+
+```python
+async Schema().execute(
+    query: str,
+    variable_values: Optional[Dict[str, Any]] = None,
+    context_value: Optional[Any] = None,
+    root_value: Optional[Any] = None,
+    operation_name: Optional[str] = None,
+    validate_queries: Optional[bool] = True,
+) -> ExecutionResult:
```

Comment:
This arrow might confuse people into thinking it's real syntax. I was thinking this was a messed up `async def` at first. Maybe we should use something like
````
```pycon
>>> .execute(...)
ExecutionResult(...)
```
````
instead?

Reply:
Yes, will try that :) I really dislike how it looks on the website at the moment, I need to find a better way to have docs for the API.